app/models/manageiq/providers/hawkular/middleware_manager/middleware_server_eap.rb

```diff
@@ -0,0 +1,4 @@
+module ManageIQ::Providers
+  class Hawkular::MiddlewareManager::MiddlewareServerEap < Hawkular::MiddlewareManager::MiddlewareServerWildfly
```

Comment:
is based on this hierarchy eap a subclass of wildfly? if yes - why?

Reply:
the requirement is that there is a general middleware_server, and it has subclasses such as eap, wildfly and in the future Fuse and others